### ISSUE TITLE
Several question about the paper

### ISSUE THREAD

**Issue by Josh00-Lu** [gwang-kim/DiffusionCLIP#16]
1) Main Paper：equation（1），it should be $\sqrt{1-\alpha _ t }w $
2) Supplement Paper : Algorithm 4, line 5, the same problem with 1)
3) Supplement Paper : Algorithm 1, line 13,14 & Algorithm 2, line 7,8. Shouldn't these $x_{ \tau _s }^{(i)}$ be with hat ?

**Comment by gwang-kim** [comment]
As I told you by e-mail, I appreciate your findings!
I'll fix them and update the arxiv submission and I will ask the chair of CVPR 2022 whether I can fix the typo.

**Comment by Josh00-Lu** [comment]
> As I told you by e-mail, I appreciate your findings! I'll fix them and update the arxiv submission and I will ask the chair of CVPR 2022 whether I can fix the typo.

Yes... Actually, this issue was opened before I mailed. Haha... Anyway, this issue can still let others know there's some mistypings in paper when reading your codes. That may reduce a lot of avoidable confusions.

**Comment by gwang-kim** [comment]
Good idea!
I reopened the issue to let other people know the erratra in our paper.
I also update the arXiv paper with the correction and request the chairs of CVPR 2022 to indicate changes.
Thank you.
